shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/domain/model/Token.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.avishek.phong.kmmic.domain.model
+
+import co.nimblehq.avishek.phong.kmmic.data.remote.model.TokenApiModel
+
+data class Token(
+    val accessToken: String,
+    val tokenType: String,
+    val expiresIn: Int,
+    val refreshToken: String,
+    val createdAt: Int
+)
+
+fun Token.toApiModel() = TokenApiModel(
```

Comment:
This `toApiModel` should be inside `TokenApiModel`. According to Clean architecture `domain` layer doesn't have access to `data` layer.

Reply:
@phongvhd93 `CheckLoggedInUseCaseImpl` inside `CheckLoggedInUseCase` is also not following the Clean architecture too. Please move `CheckLoggedInUseCaseImpl` to `data` layer instead. 🙏

Reply:
@kaungkhantsoe solved in cc86dfc

Reply:
@kaungkhantsoe `CheckLoggedInUseCaseImpl` contains the business logic and _is_ part of the domain layer. In fact, we don't need an interface for this, we can mock the concrete implementation.